zerver/webhooks/vercel/view.py

```diff
@@ -0,0 +1,52 @@
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import webhook_view
+from zerver.lib.response import json_success
+from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
+from zerver.lib.validator import WildValue, check_string
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+
+@webhook_view("Vercel")
+@typed_endpoint
+def api_vercel_webhook(
+    request: HttpRequest, user_profile: UserProfile, *, payload: JsonBodyPayload[WildValue]
+) -> HttpResponse:
+    deployment_event_name = payload["type"].tame(check_string)
+    commit_author = payload["payload"]["deployment"]["meta"]["githubCommitAuthorLogin"].tame(
```

Comment:
We should actively ignore unsupported events to avoid raising errors.
Add `IGNORED_EVENTS`, with a comment on why they're ignored, before parsing these values as they may not exist in the payload for other event types.